File: medium/2186/2186. Minimum Number of Steps to Make Two Strings Anagram II.py
class Solution:
  def minSteps(self, s: str, t: str) -> int:
    # arrs = [0] * 26
    # arrt = [0] * 26
    res = 0
    for i in range(26):
      char = chr(ord('a') + i)
      # arrs[i] = s.count(char)
      # arrt[i] = t.count(char)
      res += abs(s.count(char) - t.count(char))
    print(res)
    return res


p = Solution()
p.minSteps("cotxazilut", "nahrrmcchxwrieqqdwdpneitkxgnt")

File: medium/2186/2186. Minimum Number of Steps to Make Two Strings Anagram II.rs
// use std::num;

impl Solution {
  pub fn min_steps(s: String, t: String) -> i32 {
    let mut res = 0;

    fn count(s: &str, target: char) -> i32 {
      let mut res = 0;
      for c in s.chars() {
        res += if c as i32 == target as i32 { 1 } else { 0 };
      }

      res
    }
    for i in 0..26 {
      let c = (i + 'a' as i32) as u8 as char;
      res += (count(&s, c) - count(&t, c)).abs() as i32;
    }
    res
  }
}